.github/workflows/community.yml

```diff
@@ -11,14 +11,14 @@ on:
     types: [ created ]
 
 jobs:
-  welcome:
+  welcome: "<h1>It's great having you contribute to this project</h1> Feel free to raise an issue and create a Pull request. Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
```

Comment:
Sorry @VictoryWekwa , I did say line `14` but I made changes to the file since I wrote that message, so now it is line `20` and `21` - please can you move this to there

Reply:
Okay will make a commit on that.

Reply:
@eddiejaoude did you mean line 17?
Because that's where i see the **`welcome:`**

Reply:
This part of the yaml `welcome`, is the name of the Action, the variables for the Action are:

- `issue-message`
- `pr-message`

These variables are the ones we can keep changing and improving 🤓

Reply:
Okay. That's the part of the yaml am moving the welcome message to right?